helper.php

```diff
@@ -4,65 +4,56 @@
  */
 
 // must be run within Dokuwiki
-if (!defined('DOKU_INC')) die();
-require_once(DOKU_INC.'inc/plugin.php');
+if(!defined('DOKU_INC')) die();
 
 class helper_plugin_qc extends DokuWiki_Plugin {
 
-    function tpl(){
-        global $ACT,$INFO,$ID;
-        if (!function_exists('gd_info')) {
-            msg('You have to install php-gd lib to use the QC plugin.');
-            return;
-        }
-        if($ACT != 'show' || !$INFO['exists']) return;
-        if(p_get_metadata($ID, 'relation qcplugin_disabled')) return;
-        if ($this->getConf('adminonly')) {
-            if (!isset($_SERVER['REMOTE_USER']) || !auth_isadmin())
-                return;
-        }
+    /**
+     * Output the standard quality header. Needs to be called formt he template
+     */
+    function tpl() {
+        if(!$this->shouldShow()) return;
+
         echo '<div id="plugin__qc__wrapper">';
-        echo '<img src="'.DOKU_BASE.'lib/plugins/qc/icon.php?id='.$ID.'" width="600" height="25" alt="" id="plugin__qc__icon" />';
-        echo '<div id="plugin__qc__out" style="display:none"></div>';
+        echo '<div class="summary">';
+        echo $this->getLang('i_qcscore');
+        echo '</div>';
+        echo '<div class="qc-output"></div>';
```

Comment:
Semantically it would make probably sense to designate this as an `<aside>` instead of a `<div>`.